client/src/Components/Townbox/index.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+import './index.css'
+import {Button} from '../Button'
+
+export interface TownboxProps {
+  /**
+   * Header colour: enter either orange/primary
+   */
+  headerColor?: 'primary' | 'orange'
+
+  /**
+   * Heading contents
+   */
+  headerText?: string
+
+  /**
+   * Textbox contents
+   */
+  captionText?: string
+
+  /**
+   * Show visited button?
+   */
+  showButton?: boolean
+
+  /**
+   * specify width in terms of px
+   */
+  widthSize?: number
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Townbox: React.FC<TownboxProps> = ({
+  headerText = 'headerText',
+  captionText = 'captionText',
+  headerColor = 'orange',
+  showButton = false,
+  widthSize = 550
```

Comment:
This prop isn't needed. Ideally, we would want it to be variable length. You can define a min-width in the `tailwind.config.js` and use that